srml/support/src/runtime.rs

```diff
@@ -458,20 +481,23 @@ macro_rules! __decl_all_modules {
 				$( $parsed )*
 				$module::$name $(<$module_instance>)?,
 			};
+			{ ( $( $parsed_nested )*, $name, ) };
```

Comment:
for multiple one I can't really tell why this last comma doesn't appear, maybe cargo expand does removes it because it is equivalent